export/export.go

```diff
@@ -195,10 +195,9 @@ func (s *Service) Export(ctx context.Context) (*Resources, error) {
 //     * If UID is not nil, an error will occur. (try to find existing resource by UID)
 // (3) Get all resources. (Separated the get function to unify the struct format.)
 func (s *Service) Import(ctx context.Context, resources *ResourcesApplyConfiguration) error {
-	// TODO: Issue: #12 PR: #13
-	// if err := s.schedulerService.RestartScheduler(resources.SchedulerConfig); err != nil {
-	// 	return nil, xerrors.Errorf("restart scheduler with imported configuration: %w", err)
-	// }
+	if err := s.schedulerService.RestartScheduler(resources.SchedulerConfig); err != nil {
```

Comment:
I checked this is successful when`pluginConfig` does not exist. 
This issue may be related to #12 and #13.
Therefore, we wait for the merge of the PR.

Reply:
Okay. I think we can merge this PR without #13.
I'll review again later.